packages/stark-ui/src/modules/pretty-print/components/pretty-print.component.ts

```diff
@@ -23,6 +23,20 @@ const componentName: string = "stark-pretty-print";
  */
 const prismClassPrefix: string = "language-";
 
+/**
+ * A reference to the prettier library
+ */
+const prettier: any = require("prettier/standalone");
```

Comment:
You can install the types from Prettier (https://www.npmjs.com/package/@types/prettier) and use them here.

This will avoid issues like the one about renaming the "plugins" option in the `format()` method ;)